tensorflow_serving/servables/tensorflow/util.cc

```diff
@@ -47,6 +47,48 @@ auto* example_count_total = monitoring::Counter<1>::New(
     "/tensorflow/serving/request_example_count_total",
     "The total number of tensorflow.Examples.", "model");
 
+// Metrics by model
+auto* model_request_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_count",
+    "The total number of requests.", "model");
+
+auto* model_request_fail_count_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_fail_count",
+    "The total number of faled requests.", "model");
+
+auto* model_latency_total = monitoring::Counter<1>::New(
+    "/tensorflow/serving/model_request_latency_usec",
+    "The total time spent on executing graphs in microseconds.", "model");
+
+auto* model_latency_histogram = monitoring::Sampler<1>::New(
+    {"/tensorflow/serving/model_request_latency_histogram_usec",							    
+     "The total time spent on executing graphs in microseconds.", "model"},
+    // It would be nice to be able to set the parameters flexibly.
+    monitoring::Buckets::Explicit({
```

Comment:
can these buckets be used:

https://github.com/tensorflow/serving/blob/e206489e91a6c20b5841bf954e428870bc7b5079/tensorflow_serving/servables/tensorflow/util.cc#L65

Reply:
I have pull-requested earlier than that commit.
Oh well, I'll fix it to use this one.